STL_Extension/include/CGAL/Handle.h

```diff
@@ -71,7 +71,10 @@ class Handle
       if (PTR)
       {
         if (--PTR->count==0)
+        {
           delete PTR;
+          PTR=nullptr;
+        }
```

Comment:
What is the gain?

Reply:
Actually, the next line, after the `if`, is already `PTR=0`.

Reply:
Right, I should have looked more closely, we can probably close the PR. @efifogel is it fine with you?